### PR TITLE
PRM-656-lowercase-LPP-VATperiod Demo feedback

### PR DESCRIPTION
**Pull request by ruthdas03**

### CODE REVIEW

File: app/viewmodels/SummaryCardHelper.scala [hunk 1]
```diff
@@ -175,7 +175,7 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
 
   private def lppCardBody(lpp: LatePaymentPenalty)(implicit messages: Messages): Seq[SummaryListRow] = {
     Seq(summaryListRow(
-      messages("summaryCard.key1"),
+      messages("summaryCard.lpp.key1"),
```
Comment (LewisMitchell) [inline app/viewmodels/SummaryCardHelper.scala:178]: Checked with designers, we want to reference 'VAT period' throughout the whole service. So we could change `summaryCard.key1` in the messages file to be:
`summaryCard.key1 = VAT period`

File: app/viewmodels/SummaryCardHelper.scala [hunk 2]
```diff
@@ -195,7 +195,7 @@ class SummaryCardHelper @Inject()(link: views.html.components.link) extends Impl
     val dueDatePlus31Days: String = dateTimeToString(lpp.period.dueDate.plusDays(31))
     Seq(
         summaryListRow(
-        messages("summaryCard.key1"),
+        messages("summaryCard.lpp.key1"),
```
Comment (LewisMitchell) [inline app/viewmodels/SummaryCardHelper.scala:198]: See comment above

File: conf/messages [hunk 3]
```diff
@@ -106,6 +106,7 @@ summaryCard.addedOnKey = Added on
 summaryCard.removedReason = Reason
 summaryCard.infoOnAdjustmentPointsLinkText = Find out more about adjustment points
 summaryCard.appealCheck = Check if you can appeal
+summaryCard.lpp.key1 = VAT period
```
Comment (LewisMitchell) [inline conf/messages:109]: We could remove this if you change `summaryCard.key1` to `VAT period`

File: conf/messages.cy [hunk 4]
```diff
@@ -106,6 +106,7 @@ summaryCard.addedOnKey = Added on
 summaryCard.removedReason = Reason
 summaryCard.infoOnAdjustmentPointsLinkText = Find out more about adjustment points
 summaryCard.appealCheck = Check if you can appeal
+summaryCard.lpp.key1 = VAT period
```
Comment (LewisMitchell) [inline conf/messages.cy:109]: We could remove this if you change `summaryCard.key1` to `VAT period`

File: test/assets/messages/IndexMessages.scala [hunk 5]
```diff
@@ -105,6 +105,7 @@ object IndexMessages {
   val estimated = "estimated"
 
   val period = "VAT Period"
+  val periodLPP = "VAT period"
```
Comment (LewisMitchell) [inline test/assets/messages/IndexMessages.scala:108]: We could remove this if you change `summaryCard.key1` to `VAT period`

File: test/assets/messages/IndexMessages.scala [hunk 6]
```diff
@@ -105,6 +105,7 @@ object IndexMessages {
   val estimated = "estimated"
 
   val period = "VAT Period"
```
Comment (LewisMitchell) [inline test/assets/messages/IndexMessages.scala:107]: This will have to be 'VAT period'

**Review by LewisMitchell — CHANGES_REQUESTED** [review]
Looks good, just some changes needed

File: app/viewmodels/SummaryCardHelper.scala (hunk 1 above)
Comment (ruthdas03) [reply app/viewmodels/SummaryCardHelper.scala:178]: implemented changes in [https://github.com/hmrc/penalties-frontend/pull/127](url)

**Pull request closed without merging**